tensorboard/plugins/projector/vz_projector/vz-projector-metadata-card.html

```diff
@@ -46,6 +46,7 @@
   line-height: 24px;
   padding: 12px 12px 8px;
   width: 230px;
+  word-break: break-word;
```

Comment:
Ah! I remember why I did not use `word-break: break-word` in the past! It does not work in Firefox!
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
According to the MDN, `overflow-wrap` should work

Reply:
No problem : ) Let's check this one.